### DISCUSSION TITLE
Illuminate\Database\QueryException

### DISCUSSION THREAD

**Discussion by thyrizwan** [laravel/framework#50504] [General] [answered]
I am getting this error after running `php artisan migrate`. The project is created after Laravel 5.6 installed and using `laravel new My-Project-Name`.

1) Terminal Error Image:
![Screenshot 2024-03-13 144035](https://github.com/laravel/framework/assets/57563125/ddf75586-fac5-4e9a-8a72-af575d6304a8)

2) .env image:
![Screenshot 2024-03-13 144057](https://github.com/laravel/framework/assets/57563125/6304c0d7-03e8-4fef-bb27-adfb74d3ae9a)

3) composer.json image:
![Screenshot 2024-03-13 145428](https://github.com/laravel/framework/assets/57563125/9b3e7777-4060-46d5-a529-56aade66dad8)

**Comment by ghost** [comment]
You have to create it manually

**Reply by thyrizwan** [reply]
I already tried your solution. But I am receiving the same error.

**Reply by ghost** [reply]
Is your db user allowed to access that schema?

**Reply by thyrizwan** [reply]
Please, explain a little more.

**Reply by ghost** [reply]
Maybe this will help https://stackoverflow.com/questions/50177216/how-to-grant-all-privileges-to-root-user-in-mysql-8-0

**Reply by Raza9798** [reply]
@thyrizwan how have you installed the MySQL using any application like xampp / wampp or else directly. Please mention the MySql version on your development server

**Comment by Raza9798** [comment]
Related Issue - https://github.com/laravel/framework/issues/50526

**Comment by thyrizwan** [comment, marked as answer]
I installed latest version and problem solved. Thanks @Raza9798 for your issue suggestions.

https://github.com/laravel/framework/issues/50526

**Reply by Raza9798** [reply]
Great
